portfolio/src/main/webapp/index.html

```diff
@@ -9,34 +9,109 @@
   <body>
     <div id="content">
       <h1>Bubae's Portfolio</h1>
-      <marquee>This is Bubae's portfolio.</marquee>
-      <img src="/images/me.jpg"/>
+      <marquee>This is Bubae's portfolio. Welcome !</marquee>
+      <img src="/images/mee.jpg" class = "img1"/>
       <p>I'm Nigerian. I was born on the 23rd of July and I'm a Leo. I'm very patient and tolerant, and I don't get mad often. 
           I'm 5'5.5" and I'm dark skinned. </p>
       <p>I have worked on a robotics team at my university. I and four others made an application to aid finaicial
           literacy in the US. I am on an augmented reality research group currently creating avatars for Mozilla Hubs </p>
-      <a href = "https://www.linkedin.com/in/ebubechukwu-bubae-nwafor-ba553a181/" > LinkedIn Profile </a>
+      <a href= "https://www.linkedin.com/in/ebubechukwu-bubae-nwafor-ba553a181/" >
+        <img src="/images/linkedin.png" class = "img2" alt="LinkedIn" style="width:150px">
```

Comment:
```suggestion
        <img src="/images/linkedin.png" class = "img2" alt="LinkedIn" style="width:150px" />
```

You should add a `/` before the last `>`.